pypeit/find_objects.py

```diff
@@ -1244,6 +1244,11 @@ def global_skysub(self, skymask=None, update_crmask=True, trim_edg=(0,0),
         correction using the sky spectrum, if requested. See Reduce.global_skysub()
         for parameter definitions.
         """
+        # Skip the sky subtraction, if requested by the user
```

Comment:
It is not ideal for this to be hidden here. To make the control flow more transparent, can you move this into the routine that calls global_skysub in the main control flow.

Reply:
Yep, sounds good - done!